packages/types/src/utils.ts

```diff
@@ -83,3 +83,10 @@ type IsSerializable<T> = T extends Function ? false : true;
 export type Serializable<T> = {
   [K in keyof T as IsSerializable<T[K]> extends true ? K : never]: T[K];
 };
+
+/**
+ * Omit without union flattening
+ * */
+export type Without<T, W> = {
```

Comment:
I remember adding this in my old PR for terse params. Was it actually necessary here?

Reply:
Yeah it's an alternative to the built-in `Omit` that will omit the properties passed without doing union flattening, because we are having unions and they break if we use `Omit`.

Reply:
@dimkl This might also solve what we were discussing yesterday

Reply:
it might. i will try that out :)